test/README.md

```diff
@@ -52,8 +52,9 @@ for implementation-specific errors.
 
 ## Test Tags
 
-Some of the tests are for functionality that is not stable,
-i.e. is marked RECOMMENDED, OPTIONAL, or DRAFT.
+Some of the tests are for functionality that is optional or which might not be stable.
```

Comment:
"Might not be stable" seems a little odd -- either it's stable according to the spec or it's not, right?